app/controllers/application_controller.rb

```diff
@@ -68,11 +68,11 @@ def ensure_page
   end
 
   def set_diaspora_header
-    headers['X-Diaspora-Version'] = AppConfig.version_string
+    headers["X-Diaspora-Version"] = AppConfig.version_string
 
     if AppConfig.git_available?
-      headers['X-Git-Update'] = AppConfig.git_update if AppConfig.git_update.present?
-      headers['X-Git-Revision'] = AppConfig.git_revision if AppConfig.git_revision.present?
+      headers["X-Git-Update"] = AppConfig.git_update if AppConfig.git_update.present?
+      headers["X-Git-Revision"] = AppConfig.git_revision if AppConfig.git_revision.present?
```

Comment:
That's what I meant above - by doing those cleanup tasks as well it's harder for me, the reviewer, to actually .....review this PR.:)